react/components/Pagination/index.js

```diff
@@ -5,6 +5,8 @@ import CaretRight from '../icon/CaretRight'
 import ButtonWithIcon from '../ButtonWithIcon'
 import Dropdown from '../Dropdown'
 
```

Comment:
remove line

Reply:
I'll actually keep the blank line, along with line 3. Other components in the Styleguide also organize their imports in the General -> Specific groups separated by blank lines. I believe this adds a little bit of readability.

Reply:
all right